### ISSUE TITLE
GitHub description/readme should reflect also supporting the Microsoft Store

### ISSUE THREAD

**Issue by Sergio0694** [microsoft/winget-cli#3337]
Right now, the GitHub description says this:

> "Windows Package Manager CLI (aka winget)"

Which seems incorrect and just making the app look worse than it is. Since winget also supports installing Microsoft Store apps (or rather, installing apps that are managed by Windows Update, and not by Windows Package Manager), it really is just a general package manager that supports several sources, and not just a "Windows Package Manager CLI".

Should we update the description and readme to make this clearer?
It seems to me it'd make the app even more attractive 🙂

**Comment by Trenly** [comment]
To me it seems correct, since there isn't a GUI for winget yet - it is only a Command Line Interface. I do agree that it is a general package manager that supports several sources, and that the name "Windows Package Manager" is a bit wordy. I can certainly see how rebranding would make it more attractive, I just don't know that simply dropping the "CLI" portion would make it any more attractive to the overall userbase

**Comment by Sergio0694** [comment]
Wait, I think there's a misunderstanding here. I'm not saying the "CLI" part should be dropped, that's absolutely correct. It's the "Windows Package Manager" part that's incorrect. That is because winget is not _just_ a Windows Package Manager client, but it's also effectively a Windows Update client (since it can install Microsoft Store apps as well provided through Windows Update). So what I'm saying is that by just describing winget as "Windows Package Manager CLI", we're not making it clear that you can also use it to install any app in the Microsoft Store, thus underselling its value. Does that make sense? 🙂

As in, to me the description should be more something like:

> "A CLI package manager tool for Windows, that can find and install apps from both Windows Package Manager and the Microsoft Store"

Not saying this is necessarily the perfect description, just wanted to illustrate how this would be more accurate.
